app/models/package_manager/go.rb

```diff
@@ -205,7 +205,8 @@ def self.project_find_names(name)
       return [name] if KNOWN_VCS.any?(&name.method(:include?))
 
       begin
-        go_import = get_html("https://#{name}?go-get=1")
+        # https://go.dev/ref/mod#serving-from-proxy
+        go_import = get_html("https://#{name}?go-get=1", {request: {timeout: 2}})
```

Comment:
This was 3 years old so I looked for a better way to find this name, but couldn't find one 🤷🏻 This is at least part of the spec on how to define  a module host+name